app/javascript/dashboard/routes/dashboard/inbox/components/PriorityIcon.vue

```diff
@@ -0,0 +1,77 @@
+<script setup>
+import { CONVERSATION_PRIORITY } from 'shared/constants/messages';
+import { defineProps } from 'vue';
+
+const props = defineProps({
+  priority: {
+    type: String,
+    default: '',
+  },
+});
+
+const PRIORITY = CONVERSATION_PRIORITY;
+</script>
+
+<template>
+  <div class="inline-flex items-center justify-center rounded-md">
+    <!-- High Priority -->
+    <svg
+      v-if="props.priority === PRIORITY.HIGH"
```

Comment:
Can't we use `CONVERSATION_PRIORITY` directly here instead of creating another variable `PRIORITY`?

```suggestion
      v-if="props.priority === CONVERSATION_PRIORITY.HIGH"
```

Reply:
@muhsin-k Done